<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#login_butt{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(20, 47,78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255,255,255,210);
	border-radius:50px;
}
QPushButton#login_butt:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(40, 67,98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#login_butt:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105,118,132,200);
}

QPushButton#login_butt_2{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#login_butt:hover_2{
	color:rgba(155,168,182,220);
}
QPushButton#login_butt:pressed_2{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(115,128,142,255);
}

QPushButton#signin_butt{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(20, 47,78, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255,255,255,210);
	border-radius:50px;
}
QPushButton#sign_butt:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0., x2:1, y2:0, stop:0 rgba(40, 67,98, 219), stop:1 rgba(105, 118, 132, 226));
}
QPushButton#sign_butt:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105,118,132,200);
}

QPushButton#sign_butt_2{
	background-color: rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#signin_butt:hover_2{
	color:rgba(155,168,182,220);
}
QPushButton#signin_butt:pressed_2{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(115,128,142,255);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/prefix/bg_edited.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/prefix/bg_blurred.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>471</width>
     <height>551</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>USER</string>
    </attribute>
    <widget class="QLineEdit" name="userfield">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(,118,132,255);
color: rgb(0, 0, 0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordfield">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(,118,132,255);
color: rgb(0, 0, 0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_butt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>L O G   I N</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>NEW HERE ?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup_butt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>S I G N  U P</string>
     </property>
    </widget>
    <widget class="QLabel" name="error">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>35</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>LOG IN</string>
     </property>
    </widget>
    <zorder>signup_butt</zorder>
    <zorder>label_4</zorder>
    <zorder>login_butt</zorder>
    <zorder>error</zorder>
    <zorder>userfield</zorder>
    <zorder>passwordfield</zorder>
    <zorder>label_14</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>ADMIN</string>
    </attribute>
    <widget class="QLineEdit" name="userfield_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(,118,132,255);
color: rgb(0, 0, 0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordfield_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(,118,132,255);
color: rgb(0, 0, 0);
padding-bottom:7px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_butt_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>370</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>L O G   I N</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>35</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>LOG IN</string>
     </property>
    </widget>
    <widget class="QLabel" name="error_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>passwordfield_2</zorder>
    <zorder>userfield_2</zorder>
    <zorder>login_butt_2</zorder>
    <zorder>label_15</zorder>
    <zorder>error_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
